rust-runtime/aws-smithy-http/src/property_bag.rs

```diff
@@ -106,7 +110,12 @@ impl PropertyBag {
     pub fn get<T: Send + Sync + 'static>(&self) -> Option<&T> {
         self.map
             .get(&TypeId::of::<T>())
-            .and_then(|boxed| (&**boxed as &(dyn Any + 'static)).downcast_ref())
+            .and_then(|(_, boxed)| (&**boxed as &(dyn Any + 'static)).downcast_ref())
+    }
+
+    /// Returns an iterator of the types contained in this PropertyBag
+    pub fn contents(&self) -> impl Iterator<Item = &'static str> + '_ {
```

Comment:
Do we want this to be `pub`?

Reply:
yeah we use it in config_bag, added a note about stability